src/ralph/admin/change_list.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.admin.views.main import ChangeList, ORDER_VAR
+
+
+class RalphChangeList(ChangeList):
+    def get_ordering_from_related_model_admin(self, prefix, field_name):
+        """
+        Get ordering from related model admin.
+        """
+        field = getattr(self.model, field_name)
+        if not field:
+            return None
+        admin_site = self.model_admin.admin_site
+        model_admin = admin_site._registry[field.field.rel.to]
+        if not all([model_admin, model_admin.ordering]):
+            return None
+        return [
+            '{}{}__{}'.format(prefix, field_name, order)
+            for order in model_admin.ordering
+        ]
+
+    def get_ordering(self, request, queryset):
+        """
+        Extends ordering list by list fetching from related model admin.
+        """
+        ordering = super().get_ordering(request, queryset)
+        params = self.params
+        if ORDER_VAR in params:
+            order_params = params[ORDER_VAR].split('.')
+            for p in order_params:
+                try:
+                    none, pfx, idx = p.rpartition('-')
+                    field_name = self.list_display[int(idx)]
+                    fields = self.get_ordering_from_related_model_admin(
+                        pfx, field_name
+                    )
+                    if fields:
+                        # replace one field by list of ordering fields
+                        ordering.remove('{}{}'.format(pfx, field_name))
+                        ordering = fields + ordering
```

Comment:
what if before this loop ordering was `['a', 'b', 'c']` and here we want to replace `'b'` to `'b.1', 'b.2'`? As far as i understand this code you're changing sort order here. Shouldn't it be sth like 
```
index = ordering.index('{}{}'.format(pfx, field_name))
ordering[index:index+1] = fields
```
?